### PR TITLE
[BUGFIX ]Mark current and active state in Menu

### PR DESCRIPTION
**Pull request by linawolf**
If you open page "Theme Customization" the menu will look like this:

image

We can improve HTML and design in a follow up. 

Resolves https://github.com/TYPO3-Documentation/render-guides/issues/222

### CODE REVIEW

File: packages/typo3-docs-theme/resources/public/css/theme.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @charset "UTF-8";
 /*!
- * Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com
+ * Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com
```
Comment (brotkrueml) [inline packages/typo3-docs-theme/resources/public/css/theme.css:3]: Please run `npm ci` on the theme folder. In a previous PR I updated the node deps.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: packages/typo3-docs-theme/resources/public/css/theme.css (hunk 1 above)
Comment (linawolf) [reply packages/typo3-docs-theme/resources/public/css/theme.css:3]: this doesnt change the file. My version of fontawesome is higher then the one before for some reason

Comment (linawolf) [reply packages/typo3-docs-theme/resources/public/css/theme.css]: ok, I hope I worked it out

File: packages/typo3-docs-theme/resources/template/body/menu/mainmenu/menu-level.html.twig [hunk 2]
```diff
@@ -1,19 +1,16 @@
-{%- apply spaceless -%}
-        <li class="toctree-l{{ menuEntry.level }} {%- if menuEntry.isInRootline %} current {% endif %}">
-            <a href="{{ renderLink(menuEntry.url) }}"
-               class="nav-link {% if menuEntry.options.active %} active {% endif %}"
-                {%- if menuEntry.options.active %}  aria-current="page" {% endif -%} >
-                {{ renderNode(menuEntry.value.value) }}
-            </a>
+<li class="toctree-l{{ menuEntry.level }} {%- if menu.currentPath == entry.url %} current {%- endif -%}{% if entry.url in menu.rootlinePaths %} active {%- endif -%}">
```
Comment (brotkrueml) [inline packages/typo3-docs-theme/resources/template/body/menu/mainmenu/menu-level.html.twig:1]: Why use class "current" when no style is attached to it?

Comment (linawolf) [reply packages/typo3-docs-theme/resources/template/body/menu/mainmenu/menu-level.html.twig:1]: I think that the menu needs a new and thoughtful HTML structure anyway and thought we can do that in a follow up.

Comment (brotkrueml) [reply packages/typo3-docs-theme/resources/template/body/menu/mainmenu/menu-level.html.twig:1]: I don't like loose ends. But when you come with a follow-up in the near time, it's fine.

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**